scripts/templates/__init__.py

```diff
@@ -1,8 +1,6 @@
 """
  Copyright (C) 2023 Intel Corporation
-
  Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
  See LICENSE.TXT
  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-
```

Comment:
These 2 blank lines exist in all other files so we should keep them for consistency I think.

Reply:
yeah I accidentally deleted this, will fix